challenges/objects-arrays.js

```diff
@@ -48,8 +75,14 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 
 /* Request 1: Create a new array called universities that contains all the univeristies in the graduates array.  
 
+
 Once you have the new array created, sort the universities alphabetically and log the result. */
-const universities = [];
+
+const universities = graduates.map(universityNames).sort();
+function universityNames (names) {
+  return names.university
+}
```

Comment:
It is interesting to see you prefer to abstract you call back functions. Myself I like to inline them using fat arrows with implicit returns